Login with urbit proposal (#667)
urbit/urbit.org: content/blog/gifts-q1-2021.md

+++
title = "Gifts Q1 2021"
date = 2021-06-16
description = ""
[taxonomies]
post_type = ["gifts"]
[extra]
author = "Josh Lehman"
ship = "~wolref-podlex"
image ="https://media.urbit.org/site/posts/essays/gifts.png"
+++

<br>

<img class="ba" src="https://media.urbit.org/site/posts/essays/gifts.png">

<br>

> Twice a year we distribute address space to those that have made valuable contributions to Urbit. Pervious the rational and our thinking about the role of gifts can be seen in our previous gift announcements [here](https://urbit.org/blog/urbit-grants-and-mid-2019-gifts/) and [here](https://urbit.org/blog/gifts-q3-2020/). without further ado, here are the 2021 Q1 gift recipients (all to receive 1 star):

Reviewer: Really just fixing the typo but the sentence seemed awkward anyways so here's a suggested revision.
```suggestion
Twice a year we distribute address space to those that have made valuable contributions to Urbit. We've shared our rationale and thinking about the role of gifts in our previous gift announcements [here](https://urbit.org/blog/urbit-grants-and-mid-2019-gifts/) and [here](https://urbit.org/blog/gifts-q3-2020/).

Without further ado, here are the 2021 Q1 gift recipients (all to receive 1 star):
 ```

## Recipients

 - `~nodreb-borrus`: 	https://urbit.me/
~sitful-hatred	https://subject.network
~tocrex-holpen	https://twitter.com/tocrexholpen
~sigryn-habrex	Urbit Chess
~habsul-rignyr	The Stack Podcast
~libhut-samwes	The Stack Podcast
~tapruc-solwer	Bitcoin Wallet Consulting

> ~litmus-ritten	cultural acheivement

Reviewer 2: typo here - should be "achievement"
